### ISSUE TITLE
O Dropout e a Batch Normalization podem ser utilizados de maneira complementar nas redes neurais profundas, com o Dropout prevenindo o overfitting e a Batch Normalization acelerando o treinamento e estabilizando o aprendizado.

### ISSUE THREAD

**Issue by lucasrmagalhaes** [lucasrmagalhaes/homework-issues#1147]
- [x] Verdadeiro
